### PR TITLE
Preparing for v2 Load Balancer Migration

### PR DESCRIPTION
**Pull request by heronrs**
- New SQS stack resource 
- New convox.yml service to processes new queue

### CODE REVIEW

File: convox.yml [hunk 1]
```diff
@@ -40,6 +40,23 @@ services:
...
+    environment:
+      - AWS_REGION
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - HOST
+      - LICENSE_KEY
+      - RACK_KEY
+      - RACK_SYNC_WORKER_QUEUE
+      - TABLE_PREFIX
+    image: enterprise.convox.com/console:2.2.5
```
Comment (heronrs) [inline convox.yml:54]: I'll replace this with the new img version before merging

File: README.md [hunk 2]
```diff
@@ -209,3 +209,25 @@ Go to the "Racks" section, click the blue cog icon near your moved rack and in t
 This step is only necessary for racks you move into the console.
 
 Follow the steps in our [docs](https://docs.convox.com/management/console-rack-management/) - Moving an AWS Rack section to give the console role permission to access the EKS rack cluster.
+
+## Updating
```
Comment (Twsouza) [inline README.md:213]:
```suggestion
## Updating (V2 racks only)
```

**Review by Twsouza — APPROVED** [review]
🔥

File: README.md (hunk 2 above)
Comment (heronrs) [reply README.md:213]: These are instructions regarding the console update itself either if hosted on a v2 or v3 rack.

**Pull request merged**